[D365BC] Add note about the preview audience limited capabilities (#87912)
Azure/azure-content: articles/marketplace/dynamics-365-business-central-availability.md

Select **Save draft** before continuing to the next tab in the left-nav menu, **Technical configuration**.

> [!IMPORTANT]

> > Only the offer listing in the marketplace will be available in Preview. You will not be able to install the app in Business Central environment until you have completed the publishing process in Partner Center.

Reviewer: ```suggestion
> Only the offer listing information will be available in the marketplace preview. You won't be able to install the app in the Business Central environment until you've completed the publishing process in Partner Center.
```

Reviewer: @ktoliver I've checked with the PM team. They approved with the changes to wording here.
